contracts/contracts/core/Oracle.sol

```diff
@@ -164,8 +164,7 @@ contract Oracle is OracleStorage, IOracle, Ownable2StepUpgradeable, UUPSUpgradea
     ) private {
         if (isSlash) {
             _preconfManager.initiateSlash(
-                commitmentIndex,
-                residualBidPercentAfterDecay
```

Comment:
I guess this is ok, but I do think we should have decay for the slashed amount too

Reply:
bidder can manipulate decay parameters to skew the slashing case to be extreme, for a small bid amount.

Reply:
@chrmatt what are your thoughts?

Reply:
It's fine to have much higher slashing than bid, in fact even required for many use cases. Providers know what they are committing to when they receive the bid.